virttest/cartesian_config.py

```diff
@@ -72,7 +72,7 @@
 
     DEL -> name \\n
 
-    I^n STAT -> I^n name = VALUE | I^n name += VALUE | I^n name <= VALUE
+    I^n STAT -> I^n name = VALUE | I^n name ~= VALUE | I^n name += VALUE | I^n name <= VALUE
     I^n STAT -> I^n name ?= VALUE | I^n name ?+= VALUE | I^n name ?<= VALUE
```

Comment:
Do you mind adding a small comment clarifying these definitions? I was going through them in the past and I am not sure what exactly is meant here so any extra step to improve the docstring would be great. For instance, we can add a simple comment of a few words similarly to the ``// indentation with shift`` comment right above.

Reply:
Well, I'd recommend user to learn the grammar by reading [Cartesian Doc](https://avocado-vt.readthedocs.io/en/latest/CartesianConfig.html) instead of docstring.

We do can update these lines with:
```
I^n STAT -> I^n name = VALUE     // direct assignment
I^n STAT -> I^n name += VALUE    // addition assignment
...
```
but it is still not clear to inexperienced person to understand the behavior of each syntax, if we give a further clarification that would be another Cartesian Doc which we already have.

Reply:
Right, I didn't mean clarification on how the Cartesian config works but a clarification on how to read the special syntax above (e.g. `I^n STAT -> I^n name = VALUE` -> what is `STAT`, what is `I^n` in this context and so on in case you know). Using the double-doc logic you stated we could also remove these lines if they are unclear and nobody knows what they mean or add an extra line or two stating what the syntax is about. Note that I don't mean the Cartesian config syntax which is clear, but the syntax used just within these lines in the docstring.

Reply:
I see, but sorry I don't quite understand about the token `I^n` and `STAT` neither, so let's leave this stuff to someone knows that.

Reply:
Sure, I only suggest this if you are aware of the meaning of these lines. Let's leave it from someone that knows more then - thanks for updating these lines nevertheless though.